File: abc/abc305/b.cpp
// https://atcoder.jp/contests/abc305/submissions/42128511

#include <bits/stdc++.h>
using namespace std;

int main() {
  vector<int> ptn = {0, 3, 4, 8, 9, 14, 23};

  char p, q;
  cin >> p >> q;

  int pp = (int)(p - 'A');
  int qq = (int)(q - 'A');
  if(pp > qq) swap(pp, qq);

  cout << ptn[qq] - ptn[pp];
}

File: abc/abc305/b.java
// https://atcoder.jp/contests/abc305/submissions/43749103

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int p = (int)(stdIn.next().charAt(0) - 65);
    int q = (int)(stdIn.next().charAt(0) - 65);

    int[] a = {0, 3, 1, 4, 1, 5, 9};
    int[] b = new int[7];
    for(int i = 0; i < 7; i++) {
      if(i == 0) {
        b[i] = 0;
      } else {
        b[i] = b[i - 1] + a[i];
      }
    }

    if(p > q) {
      int t = p;
      p = q;
      q = t;
    }

    System.out.println(b[q] - b[p]);
  }
}
